frontend/public/components/volumes-table.tsx

```diff
@@ -131,7 +132,12 @@ const VolumesTableRows = ({ componentProps: { data } }) => {
         },
       },
       {
-        title: readOnly ? i18next.t('public~Read-only') : i18next.t('public~Read/Write'),
+        title:
+          readOnly ||
+          podVolume?.persistentVolumeClaim?.readOnly === 'true' ||
+          podVolume?.nfs?.readOnly === 'true'
```

Comment:
If you look at [VolumeSource](https://github.com/kubernetes/kubernetes/blob/d89d5ab2680bc74fe4487ad71e514f4e0812d9ce/staging/src/k8s.io/api/core/v1/types.go#L49) type definition, `nfs` is not the only item.

In OCP we support at least `fc`, `iscsi`, `flexVolume` and `csi` that can have `.readOnly` flag. It would be great if you could detect `podVolume.*.readOnly` exists.

Reply:
Thanks, @jsafrane.  Unfortunately there is no slick way to use a wildcard in JavaScript, so I've enumerated all the options.

Reply:
> Thanks, @jsafrane. Unfortunately there is no slick way to use a wildcard in JavaScript, so I've enumerated all the options.

Disregard.  @spadgett had a solution.  Updated.

Reply:
I don't understand javascript enough to review that piece of code, I trust you it detects that `podVolume.*.readOnly` exists